__init__.py

```diff
@@ -29,21 +29,23 @@
 
 from neon_utils.skills.neon_skill import NeonSkill, LOG
 from mycroft.skills.core import intent_file_handler
-from .request_handling import RequestHandler
-from .request_handling import existing_lang_check, get_shop_data,\
-                                shop_selection_by_floors,\
-                                location_format,\
-                                curent_time_extraction
+from .request_handling import existing_lang_check, get_store_data, \
+    store_selection_by_floors, \
+    location_format, \
+    curent_time_extraction
 import re
+import lingua_franca
+from lingua_franca.format import nice_duration
 
+lingua_franca.load_language('en')
```

Comment:
This shouldn't happen at import time; if needed for tests, please move to test init